### PR TITLE
inspectrum: add application bundle and cleanup

### PR DESCRIPTION
**Pull request by ra1nb0w**
#### Description

- adding application bundle to simplify the usage;
  there is not official icon therefore I took a wallpaper; this file
  is provided without license.
- cleanup to be compliant with `port lint`

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on
macOS 10.15 19A582a
Xcode 11.2 11B41

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by kencu** [comment]
that link back should not be needed; the app PG is doing everything in the background.

**Comment by kencu** [comment]
I thought you should only have to spec includes and libs if they are in nonstandard locations, so I suspect those configure args are not needed.

**Comment by michaelld** [comment]
> that link back should not be needed; the app PG is doing everything in the background.

Could be ... since `destroot` failed, I didn't get to see what the .app PG does in this regard ...

**Comment by michaelld** [comment]
> I thought you should only have to spec includes and libs if they are in nonstandard locations, so I suspect those configure args are not needed.

It's important to provide cmake specs for packages with possible duplicate install locations. Python is the top of the list ... always! We could probably do away with the Liquid-DSP args ... but I'd be wary of removing the FFTW3 args.

That said: If the user is (gasp) trying to use another package manager alongside MacPorts, we're much better off telling CMake what to do than to rely on it to find things itself, as it's prone to finding the API from one package and the ABI from another ... or, more broadly, mixing and matching package offerings without knowing the difference ... which generally isn't a good or wise thing to do & just leads to confusion & challenges for us Portfile developers. It's a relatively minimal cost to tell CMake what do to, both in terms of looking up the correct variables as well as the amount of Portfile space. IMHO, of course ...

**Comment by michaelld** [comment]
Looking at the `limesuite` Portfile, it does indeed look like the .app PG automatically does the linking between `${prefix}/bin/NAME` to ${applications_dir}/NAME.app/Contents/MacOS/NAME` ... so @ra1nb0w you can ignore my #2 change request. Guessing if the missing file is provided that'll take care of my #1 change request ... which should then allow the port to install correctly. Looking forward to this change!

**Comment by ra1nb0w** [comment]
(1) strange, because the patch is correct and contains `inspectrum/files/inspectrum.icns`
Have you used `git am` to apply the patch? because that format doesn't work with patch and binary.

About `configure.args-append`: generally I don't use it if they are not in strange position or there is `select` therefore, if you like, I can remove the lines.

**Comment by michaelld** [comment]
OK ... yes that's strange I see the `.icns` file too ... let me try again ...

**Comment by michaelld** [comment]
So the issue is when I look at `5431.patch`, the patchfile contains the `.icns` but the file isn't created for some reason. My bad for not being thorough enough the first time around! Let me figure out a way to make this patch work ...
